src/main/java/homework05/Type.java

```diff
@@ -0,0 +1,20 @@
+package homework05;
+
+public enum Type {
+	APPLE("Italy"), STRAWBERRY("Poland"), PEAR("Ukraine"), ORANGE("Spain"), PINEAPPLE("Turkey"), BANANA("Ecuador");
+
+	String country;
```

Comment:
private

Reply:
I use country in toString of Fruit class...